index.html

```diff
@@ -45,6 +45,170 @@ <h2 class="secondary-titel">
+          <img class="project-img" src="#" alt="project2-image">
+          <div class="project-titel-container">
+            <h4 class="project-titel">Covid 19 Web app</h4>
+          </div>
+          <div class="tags-container">
+            <div class="html-tag tags">html</div>
+            <div class="css-tag tags">css</div>
+            <div class="javascript-tag tags">JavaScript</div>
+            <div class="react-tag tags">React</div>
+          </div>
+          <form action="#" method="get">
+            <button type="submit" class="project-btn">See Project</button>
+          </form>
+        </div>
+        <div class="project3 projects-cards">
+          <img class="project-img" src="#" alt="project3-image">
+          <div class="project-titel-container">
+            <h4 class="project-titel">Face Detection Web app</h4>
+          </div>
+          <div class="tags-container">
+            <div class="html-tag tags">html</div>
+            <div class="css-tag tags">css</div>
+          </div>
+          <form action="#" method="get">
+            <button type="submit" class="project-btn">See Project</button>
+          </form>
+        </div>
+        <div class="project4 projects-cards">
+          <img class="project-img" src="#" alt="project4-image">
+          <div class="project-titel-container">
+            <h4 class="project-titel">Project 4</h4>
+          </div>
+          <div class="tags-container">
+            <div class="html-tag tags">Html</div>
+            <div class="css-tag tags">CSS</div>
+          </div>
+          <form action="#" method="get">
+            <button type="submit" class="project-btn">See Project</button>
+          </form>
+        </div>
+        <div class="project5 projects-cards">
+          <img class="project-img" src="#" alt="project5-image">
+          <div class="project-titel-container">
+            <h4 class="project-titel">Project 5</h4>
+          </div>
+          <div class="tags-container">
+            <div class="html-tag tags">html</div>
+            <div class="css-tag tags">css</div>
+          </div>
+          <form action="#" method="get">
+            <button type="submit" class="project-btn">See Project</button>
+          </form>
+        </div>
+        <div class="project1 projects-cards">
+          <img class="project-img" src="#" alt="project6-image">
+          <div class="project-titel-container">
+            <h4 class="project-titel">Project 6</h4>
+          </div>
+          <div class="tags-container">
+            <div class="html-tag tags">html</div>
+            <div class="css-tag tags">css</div>
+          </div>
+          <form action="#" method="get">
+            <button type="submit" class="project-btn">See Project</button>
+          </form>
+        </div>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="main-about-me-container">
+        <h1 class="about-me-titel">About me</h1>

```

Comment:
- [ ] The `<h1>` tag is already in use please us please us a header tag but with another number.